notebooks/learning-to-rank/01-learning-to-rank.ipynb

```diff
@@ -0,0 +1,1556 @@
+        "\n",
+        "from tqdm import tqdm\n",
+        "# Setup the progress bar so we can use progress_apply in the notebook.\n",
+        "tqdm.pandas()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yks44hf0mOpI"
+      },
+      "source": [
+        "## Configure your Elasticsearch deployment\n",
+        "\n",
+        "For this example, we will be using an [Elastic Cloud](https://www.elastic.co/guide/en/cloud/current/ec-getting-started.html) deployment (available with a [free trial](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook))."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 71
+        },
+        "id": "IpnP7JUHmOpI",
+        "outputId": "eb52c692-a773-4863-f930-fdedb5c6e0eb"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "'Successfully connected to cluster runTask (version 8.13.0-SNAPSHOT)'"
+            ]
+          },
+          "execution_count": 5,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "import getpass\n",
+        "from elasticsearch import Elasticsearch\n",
+        "\n",
+        "# Found in the \"Manage Deployment\" page\n",
+        "try: CLOUD_ID\n",
+        "except NameError: CLOUD_ID = getpass.getpass(\"Enter Elastic Cloud ID: \")\n",
+        "\n",
+        "# Password for the \"elastic\" user generated by Elasticsearch\n",
+        "try: ELASTIC_PASSWORD\n",
+        "except NameError:\n",
+        "    ELASTIC_PASSWORD = getpass.getpass(\"Enter Elastic password: \")\n",
+        "\n",
+        "# Create the client instance\n",
+        "es_client = Elasticsearch(\n",
+        "    cloud_id=CLOUD_ID,\n",
+        "    basic_auth=(\"elastic\", ELASTIC_PASSWORD)\n",
+        ")\n",
+        "\n",
+        "client_info = es_client.info()\n",
+        "\n",
+        "f\"Successfully connected to cluster {client_info['cluster_name']} (version {client_info['version']['number']})\""
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KLAN6aq_mOpJ"
+      },
+      "source": [
+        "## Configuring the dataset\n",
+        "\n",
+        "In this example notebook we will use a dataset derived from [MSRD](https://github.com/metarank/msrd/tree/master) (Movie Search Ranking Dataset).\n",
+        "\n",
+        "The dataset is available [here](https://github.com/elastic/elasticsearch-labs/tree/main//ltr-notebook/notebooks/learning-to-rank/sample_data/) and contains the following files:\n",
+        "\n",
+        "- **movies_corpus.jsonl.gz**\n",
+        "- **movies_judgements.csv.gz**:\n",
+        "- **movies_index_settings.json**"
```

Comment:
Can you add a short description to each of these files/datasets? Just explaining shortly what they are. I know it's pretty obvious from the title but maybe something like:

* movies_corpus: The movies dataset which will be indexed.
* movies_judgements: A file containing relevance judgments for a set of queries

Reply:
Added descriptions.